client/task-list/tasks/tax.js

```diff
@@ -508,6 +520,8 @@ export default compose(
 			[ 'jetpack', 'woocommerce-services' ],
 			activePlugins
 		);
+		window.activePlugins = activePlugins;
```

Comment:
Are these two lines needed or perhaps some debugging code?

Reply:
@timmyc My apologies and thank you for catching. Yes, they were for debugging. I will remove them.

Reply:
Removed from the latest commit https://github.com/woocommerce/woocommerce-admin/commit/27330e602118a08e4548ce9132e00d7a873290a3